dags/myla311/dag-311-to-arc.py

```diff
@@ -0,0 +1,96 @@
+import pandas as pd
+import arcgis 
+from arcgis.gis import GIS
+from arcgis.features import FeatureLayerCollection
+import geopandas as gpd 
+import os 
+import shutil
+import pathlib
+
+from airflow import DAG
+from airflow.operators.bash_operator import BashOperator
+from airflow.operators.python_operator import PythonOperator
+from datetime import datetime, timedelta
+
+def overwrite_layer(**kwargs):
+    """
+    Overwrites the layer 
+    on a ESRI GIS
+    """
+    print('logging into ESRI')
+
+    gis = GIS("http://lahub.maps.arcgis.com/home/index.html", 
+            os.environ.get('LAHUB_USERNAME'),
+            os.environ.get('LAHUB_PASSWORD'))
+    # get the item 
+
+    my_content = gis.content.search(query="311_test owner:" + gis.users.me.username, 

```

Comment:
Is there a better way to find this item rather than through a search?

Reply:
done by item id now.